sbvr-compiler-libs.js

```diff
@@ -20,22 +20,24 @@
     };
     SBVRCompilerLibs.TYPE_VOCAB = "Type";
     SBVRCompilerLibs.IsPrimitive = function(term) {
-        return term[2] == this.TYPE_VOCAB ? term[1] : (term = this.FollowConceptType(term)) !== !1 && term[2] == this.TYPE_VOCAB ? term[1] : !1;
+        return term[2] == this.TYPE_VOCAB ? term[1] : !1 !== (term = this.FollowConceptType(term)) && term[2] == this.TYPE_VOCAB && term[1];
     };
     SBVRCompilerLibs.IsChild = function(child, parent) {
-        do if (this.IdentifiersEqual(child, parent)) return !0; while ((child = this.FollowConceptType(child)) !== !1);
```

Comment:
Wow, that was ugly!

Reply:
I mean it is called uglifyjs for a reason :P

Reply:
Yeah, it definitely lives up to its name :)